internal/pkg/template/templates/workloads/partials/cf/nlb.yml

```diff
@@ -66,12 +67,33 @@ NLBTargetGroup:
       Fn::ImportValue:
         !Sub "${AppName}-${EnvName}-VpcId"
 
+NLBSecurityGroup:
+  Metadata:
+    'aws:copilot:description': 'A security group for your network load balancer to route traffic to service'
+  Type: AWS::EC2::SecurityGroup
+  Properties:
+    GroupDescription: Allow access from the network load balancer to service
+    SecurityGroupIngress:
+{{range $cidr := .NLB.PublicSubnetCIDRs}}
+      - CidrIp: {{$cidr}}
```

Comment:
cool! one thing that we can do in our e2e test is to write a test that ensures that if I tried to access a task IP address from the public internet without going through NLB the request will be blocked.

This will help us feel good about security